### PR TITLE
Adding contributor guides for 5 charts

### PR DESCRIPTION
**Pull request by srmukher**
Adding contributor guides for 5 charts:
1. Donut chart
2. Vertical bar chart
3. Grouped vertical bar chart
4. Vertical Stacked bar chart
5. Sparkline chart

### CODE REVIEW

File: docs/Charting-Concepts/VerticalBarChart.md [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+   1. Work item: <https://uifabric.visualstudio.com/iss/_workitems/edit/7434>
+- **Accessibility**
+  Vertical bar chart is tested for fast pass accessibility.
+  Link to the FAST pass tool: <https://accessibilityinsights.io/docs/web/getstarted/fastpass/> 
+  Our charts have elaborate accessibility support. The charts are WCAG 2.1 MAS C compliant for accessibility.
+  Consumers can define their own aria labels for each point by setting the callOutAccessibilityData property.
+- **Theming**
+  The palette for vertical bar chart is set from the "theme" prop as passed to the component during rendering. Both light and dark themes are supported and users can create there own theme too. [Ref6](https://github.com/microsoft/fluentui/wiki/Theming)  and [Ref7](https://github.com/microsoft/fluentui/wiki/How-to-apply-theme-to-Fluent-UI-React-components)  explains theming in detail.
+- **Debugging**
+  The detailed steps on debugging has been given in [Debugging](https://uifabric.visualstudio.com/iss/_wiki/wikis/iss.wiki?wikiVersion=GBwikiMaster&pagePath=/Fluent%20React%20Charting/Debugging).
```
Comment (AtishayMsft) [inline docs/Charting-Concepts/VerticalBarChart.md:322]: Update the links pointing to wiki to contrib site

File: docs/Charting-Concepts/VerticalStackedBarChart.md [hunk 2]
```diff
@@ -0,0 +1,528 @@
...
+    ```
+    import { render, screen, queryAllByAttribute } from '@testing-library/react';
+
+    import { chartPoints } from './VerticalBarChart.test';
+    ```
+- Certain props need async await structure (waitFor in react testing library) for different props or nested SVGs to render.
+- **Design figma**
+  Vertical Stacked Bar Chart Figma - [Link](https://www.figma.com/file/WOoCs0CmNYZhYl9xXeCGpi/Data-viz-\(Archive\)?type=design&node-id=21153-80197&mode=design&t=kjtgBpG3tOGCcx00-0) 
+- **Known issues**
+  - The axis titles are to be added (Already in [PR](https://github.com/microsoft/fluentui/pull/28580))
```
Comment (AtishayMsft) [inline docs/Charting-Concepts/VerticalStackedBarChart.md:430]: This is implemented now. Make updates

Comment (srmukher) [reply docs/Charting-Concepts/VerticalStackedBarChart.md]: updated.

File: docs/Charting-Concepts/VerticalBarChart.md (hunk 1 above)
Comment (srmukher) [reply docs/Charting-Concepts/VerticalBarChart.md]: updated the debugging link which is available in the contrib repo

File: docs/Charting-Concepts/GroupedVerticalBarChart.md [hunk 3]
```diff
@@ -0,0 +1,522 @@
...
+- **Theming**
+  The palette for vertical bar chart is set from the "theme" prop as passed to the component during rendering. Both light and dark themes are supported and users can create there own theme too. [Ref6](https://github.com/microsoft/fluentui/wiki/Theming)  and [Ref7](https://github.com/microsoft/fluentui/wiki/How-to-apply-theme-to-Fluent-UI-React-components)  explains theming in detail.
+- **Debugging**
+  Following are some of the ways in which the donut chart code can be debugged:
+  - *Using logging*: Console logs can be used to debug the different variables within the components.
+  - *Using Browser debugger*: Putting breakpoints in the source code in browser and running the debugger.
+  - *Using Extension*: For debugging a test in VSCode, the launch.json contains the configurations for debugging the current open test. We can directly put breakpoints in the current open test and run the debugger.
+- **Variants**
+  Following are the variants of vertical bar chart: [Ref8](https://fluentuipr.z22.web.core.windows.net/heads/master/react-charting/demo/index.html#/examples/VerticalBarchart)
```
[the remark this answers is not in the document]
Comment (srmukher) [reply docs/Charting-Concepts/GroupedVerticalBarChart.md]: done

File: docs/Charting-Concepts/VerticalBarChart.md [hunk 4]
```diff
@@ -0,0 +1,449 @@
...
+1. D3-scale:
+   - [Link 1](https://github.com/d3/d3-scale/blob/main/README.md)
+   - [Link 2](https://d3js.org/d3-scale)
+2. [D3-shape](https://github.com/d3/d3-shape/blob/main/README.md)
+3. [D3-selection](https://github.com/d3/d3-selection/blob/main/README.md)
+4. [D3-array](https://github.com/d3/d3-array/blob/main/README.md)
+5. [D3-axis](https://github.com/d3/d3-axis/blob/main/README.md)
+6. [How to apply theme](https://github.com/microsoft/fluentui/wiki/How-to-apply-theme-to-Fluent-UI-React-components)
+7. [Theming](https://github.com/microsoft/fluentui/wiki/Theming)
+8. [Vertical Bar Chart](https://fluentuipr.z22.web.core.windows.net/heads/master/react-charting/demo/index.html#/examples/VerticalBarchart)
```
[the remark this answers is not in the document]
Comment (AtishayMsft) [reply docs/Charting-Concepts/VerticalBarChart.md:410]: I still see this at multiple places

**Review by AtishayMsft — APPROVED** [review]

**Pull request merged**